CMakeLists.txt

```
@@ -58,6 +58,7 @@ add_library(${PROJECT_NAME}
   src/quat.c
   src/cam.c
   src/vec2.c
+  src/vec2.c
```

This duplicates the line above
```suggestion
  src/vec2.c
```